src/api/metadataApi.ts

```diff
@@ -25,21 +25,25 @@ const metadataApi = {
 
     const { retryInterval } = newOptions;
     let { retries } = newOptions;
+    let metaData: null | unknown; // XXX: Implement the full description of the type here
```

Comment:
Why not do it in this PR?

Reply:
Added the metadata type as received from the explorer service data source.